### PR TITLE
JDI homework done

### PR DESCRIPTION
**Pull request by PIDAMI**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw/jdi/data/provider/FormDataProvider.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.epam.tc.hw.jdi.data.util.MetalsColorsJsonReader;
+import org.testng.annotations.DataProvider;
+
+public class FormDataProvider {
+
+    @DataProvider(name = "metalsColorsFormData")
+    public static Object[][] metalsColorsFormData() {
+        return MetalsColorsJsonReader.readData()
+                                     .stream()
```
Comment (alstafeev) [inline src/test/java/com/epam/tc/hw/jdi/data/provider/FormDataProvider.java]: large margin

File: src/test/java/com/epam/tc/hw/jdi/steps/AssertStep.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import com.epam.tc.hw.jdi.entities.MetalsColorsFormEntry;
+import com.epam.tc.hw.jdi.uiobjects.site.pages.MetalsColorsPage;
+import io.qameta.allure.Step;
+import java.util.List;
+
+public class AssertStep {
+    @Step
+    public static void checkUserIsLoggedIn() {
+        assertThat(userName.isDisplayed()).isTrue();
```
Comment (alstafeev) [inline src/test/java/com/epam/tc/hw/jdi/steps/AssertStep.java]: you should use jdi assert

**Review by alstafeev — CHANGES_REQUESTED** [review]
for this class and for the whole project, check that it is in the same style.
you have a blank line before the first line in the method and don't have blank string before first line as well.

File: src/test/java/com/epam/tc/hw/jdi/data/provider/FormDataProvider.java (hunk 1 above)
Comment (PIDAMI) [reply src/test/java/com/epam/tc/hw/jdi/data/provider/FormDataProvider.java]: split `MetalsColorsJsonReader.readData()` to separate lines so margin's smaller now

File: src/test/java/com/epam/tc/hw/jdi/steps/AssertStep.java (hunk 2 above)
Comment (PIDAMI) [reply src/test/java/com/epam/tc/hw/jdi/steps/AssertStep.java]: changed to `userName.is().displayed()`

**Review by alstafeev — APPROVED** [review]
You should resolve merge conflicts

**Pull request merged**